packages/docs/src/styles/components/syntaxHighlighting.scss

```diff
@@ -11,10 +11,10 @@ pre {
 }
 
 code {
-  display: inline-block;
```

Comment:
Removed this so the inline code wouldn't result in periods wrapping to the next line at certain breakpoints